qemu/hw/arm/tosa.c

```diff
@@ -19,9 +19,25 @@
 
 static int tosa_init(struct uc_struct *uc, MachineState *machine)
 {
-    if (uc->mode & UC_MODE_MCLASS) {
+
+    if (uc->mode & UC_MODE_MCLASS)
```

Comment:
for if block, please put { at the end of the same line, but not in a new line.

Reply:
Would omitting braces for these one-line `if` cases be preferable, or do you specifically just want braces on the same line in all cases?

Reply:
Thanks, see  da417e8